src/main/java/com/google/sps/Scheduler.java

```diff
@@ -85,7 +85,7 @@ private void generateSchedulesHelper(List<Course> availableCourses, List<Schedul
 
     List<Section> scheduleSections = new ArrayList<>();
     for (ScheduledCourse c : currentSchedule) {
-      scheduleSections.add(c.getLectureSection());
+      scheduleSections.add(c.getlectureSection());
```

Comment:
Don't forget to adjust this to be `getLectureSection` instead of `getlectureSection`

Reply:
Thanks for the catch. Pretty sure its updated in a different commit, I'll check it out.